BUILD.bazel

```diff
@@ -993,13 +1001,18 @@ ray_cc_library(
         ],
     }),
     deps = [
-        ":node_manager_fbs",
+        ":raylet_client_connection_lib",
         ":node_manager_rpc",
-        ":ray_common",
+        "//src/ray/common:id",
+        "//src/ray/common:asio",
+        "//src/ray/common:ray_object",
+        "//src/ray/common:status",
         "//src/ray/common:network",
+        "//src/ray/common:task_common",
+        "//src/ray/util:util",
```

Comment:
Checking the code, I don't think it's a necessary dependency (also could remove it from header inclusion)

Reply:
Removed.